apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.sql

```diff
@@ -0,0 +1,101 @@
+-- BLOCK select_assessment_access_policies
+SELECT
+  format_date_full_compact (
+    aap.created_at,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS created_at,
+  aap.created_by::text AS created_by,
+  aap.credit::text AS credit,
+  format_date_full_compact (
+    aap.end_date,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS end_date,
+  aap.note AS note,
+  format_date_full_compact (
+    aap.start_date,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS start_date,
+  (
+    SELECT
+      name
+    FROM
+      groups
+    WHERE
+      id = aap.group_id
+  ) AS group_name,
+  (
+    SELECT
+      uid
+    FROM
+      users
+    WHERE
+      user_id = aap.user_id
+  ) AS student_uid,
+  aap.id AS id
+FROM
+  assessment_access_policies AS aap
+  JOIN assessments AS a ON (a.id = aap.assessment_id)
+  JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
+  JOIN pl_courses AS c ON (c.id = $course_id)
```

Comment:
No need to pass in a course ID, you can get it by doing a join with the course instance.

```suggestion
  JOIN pl_courses AS c ON (c.id = ci.course_id)
```

Reply:
fixed in [92cbd0c](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/92cbd0c103797f07ae80ace8887d7ca4ae0f306a)